src/modules/src/extrx.c

```diff
@@ -177,14 +177,42 @@ static void extRxDecodeSpektrum(void)
 
 /* Loggable variables */
 #ifdef ENABLE_EXTRX_LOG
+/**
+ * External receiver (RX) log group. This contains received raw
+ * channel data as well as RPTY data after it has been converted.
```

Comment:
What is RPTY?

Reply:
roll pitch trust yaw I guess? It's for attitude control

Reply:
Roll Pitch Yaw Thrust is my guess